x11-misc/compiz-debug-utils/compiz-debug-utils-0_pre20171023.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+myCommit="f700dd2b6af1f0c93b6b1eb1d75e658783f848e2"
+
+DESCRIPTION=" Helper utilities for debugging Compiz"
```

Comment:
+DESCRIPTION="Helper utilities for debugging Compiz"

Reply:
I think @jonasstein means to say you've got extra space here.